### PR TITLE
Cleaner removewordslist

### PR DESCRIPTION
**Pull request by duramato**
- [ ] PR is based on the DEVELOP branch
- [ ] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [ ] Read [contribution guide](https://github.com/PyMedusa/SickRage/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by fernandog** [comment]
Approved

**Comment by fernandog** [comment]
TODO: make it work for this one: .www.Cpasbien.pw

**Comment by fernandog** [comment]
@duramato @labrys need to double check:

https://regex101.com/r/wW0zP4/5

tested with:
.[www.usabit.com]
[www.Cpasbien.pe] 
[www.Cpasbien.com] 
[ www.Cpasbien.pw ] 
.www.Cpasbien.pw nok
[www.newpct1.com]
[ www.Cpasbien.com ] 
 - { www.SceneTime.com }
{ www.SceneTime.com } - 
].[www.tensiontorrent.com] - 
].[ www.tensiontorrent.com ] - 
 - [ www.torrentday.com ]
[ www.TorrentDay.com ] -

File: sickbeard/helpers.py [hunk 1]
```diff
@@ -132,18 +128,9 @@ def remove_non_release_groups(name):
...
-        r'\.\(NLsub\)$': 'searchre',
         r'\(NLsub\)$': 'searchre',
-        r'\.\[BT\]$': 'searchre',
         r' \[1044\]$': 'searchre',
-        r'\.RiPSaLoT$': 'searchre',
-        r'\.GiuseppeTnT$': 'searchre',
-        r'\.Renc$': 'searchre',
-        r'\.gz$': 'searchre',
-        r'\.English$': 'searchre',
-        r'\.German$': 'searchre',
```
Comment (OmgImAlexis) [inline sickbeard/helpers.py]: Is there a reason all of these got removed?

Comment (duramato) [inline sickbeard/helpers.py]: They are absolved by the new ones..

**Comment by OmgImAlexis** [comment]
Approved!

**Pull request merged**